src/AutoRest.CSharp/Common/Output/Expressions/ValueExpressions/PositionalParameterReference.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+namespace AutoRest.CSharp.Common.Output.Expressions.ValueExpressions
+{
+    internal record PositionalParameterReference(string ParameterName, ValueExpression ParameterValue) : ValueExpression;
```

Comment:
What's positional parameter?

Reply:
when you have a method:
```csharp
public void DoSomething(string foo, string bar = null) {}
```
when you call it, you could use this:
```csharp
DoSomething(bar: "a", foo: "b")
```
the expression `bar: "a"` here is called a "positional parameter argument"